slo/operators.py

```diff
@@ -215,16 +216,22 @@ def _eval(self, y, x, alpha=1, beta=0, forward=True):
         X = x.reshape( self._ft_shape + (x.shape[1],) )
         Y = y.reshape( self._ft_shape + (x.shape[1],) )
 
-        u,v,w,batch = X.shape
-        nflops = batch * 5 * u*v*w * np.log2(u*v*w)
-        nbytes = X.nbytes * 2 + Y.nbytes * 2
-        nthreads = self._backend.get_max_threads()
+        batch = X.shape[3]
+        bs = self._batch_size if self._batch_size != None else batch
+        for b in range(0, batch, bs):
+            u,v,w,batch = X.shape
+            nflops = batch * 5 * u*v*w * np.log2(u*v*w)
```

Comment:
nflops should reflect the number of flops performed by a single `backend.fftn` call, so I think it should be `(batch_max - b) * 5 * ...`.